.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,26 +3,26 @@ name: Bug report
 about: Something works incorrectly or doesn't work at all
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Describe the Bug**
+A clear and concise description of what the bug is. -->
```

Comment:
`-->`?

Reply:
I thought about wrapping the section descriptions in HTML comments but decided against it, but accidentally left that in. Whoosp. Fixed it. :^)